hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/key/TestOMKeyCreateRequest.java

```diff
@@ -629,6 +631,29 @@ public void testKeyCreateWithFileSystemPathsEnabled() throws Exception {
 
   }
 
+  @Test
+  public void testPreExecuteWithInvalidKeyPrefix() throws Exception {
+
+    KeyArgs.Builder keyArgs = KeyArgs.newBuilder()
+        .setVolumeName(volumeName).setBucketName(bucketName)
```

Comment:
Similarly add a test case for valid keys as well

Reply:
yup added!